haystack/utils/import_utils.py

```diff
@@ -93,6 +95,20 @@ def load_documents_from_hf_datasets(dataset_name: str, split: Optional[str] = "t
     return documents
 
 
+def get_filename_extension_from_url(url: str) -> Tuple[str, str]:
+    """
+    Extracts the filename and file extension from an url.
+
+    :param url: http address
+    :return: Tupel (filename, file extension) of the file at the url.
+    """
+    parsed = urlparse(url)
+    root, extension = splitext(parsed.path)
+    archive_extension = extension[1:]
+    file_name = unquote(basename(root[1:]))
+    return (file_name, archive_extension)
```

Comment:
you can skip the parentheses
```suggestion
    return file_name, archive_extension
```